Dockerfile

```diff
@@ -1,27 +1,40 @@
 FROM lwthiker/curl-impersonate:0.5-ff-slim-buster AS curlimpersonate
 
-FROM php:8.0.27-fpm-buster AS rssbridge
+FROM debian:12-slim AS rssbridge
 
 LABEL description="RSS-Bridge is a PHP project capable of generating RSS and Atom feeds for websites that don't have one."
 LABEL repository="https://github.com/RSS-Bridge/rss-bridge"
 LABEL website="https://github.com/RSS-Bridge/rss-bridge"
 
+ARG DEBIAN_FRONTEND=noninteractive
 RUN apt-get update && \
     apt-get install --yes --no-install-recommends \
+      ca-certificates \
       nginx \
-      zlib1g-dev \
-      libzip-dev \
-      libmemcached-dev \
+      php-curl \
+      php-fpm \
+      php-intl \
+      # php-json is enabled by default with PHP 8.2 in Debian 12
+      php-mbstring \
+      php-memcached \
+      # php-opcache is enabled by default with PHP 8.2 in Debian 12
+      # php-openssl is enabled by default with PHP 8.2 in Debian 12
+      php-sqlite3 \
+      php-xml \
+      php-zip \
+      # php-zlib is enabled by default with PHP 8.2 in Debian 12
       nss-plugin-pem \
-      libicu-dev && \
-    docker-php-ext-install zip && \
-    docker-php-ext-install intl && \
-    pecl install memcached && \
-    docker-php-ext-enable memcached && \
-    docker-php-ext-enable opcache && \
-    mv "$PHP_INI_DIR/php.ini-production" "$PHP_INI_DIR/php.ini"
-
-COPY ./config/nginx.conf /etc/nginx/sites-enabled/default
+      && \
+    rm -rf /var/lib/apt/lists/*
+
+COPY ./config/nginx.conf /etc/nginx/sites-available/default
+COPY ./config/php-fpm.conf /etc/php/8.2/fpm/pool.d/rss-bridge.conf
+COPY ./config/php.ini /etc/php/8.2/cli/conf.d/90-rss-bridge.conf
```

Comment:
this php.ini is ONLY used by cli. in case you didnt see that

Reply:
Sorry, well spotted

Reply:
https://github.com/RSS-Bridge/rss-bridge/pull/3500/commits/fb01d28260d7378b726013789a3b20d91bbeb08b